src/www/ui/admin-license-file.php

```diff
@@ -57,6 +57,7 @@ function RegisterMenus()
   public function Output()
   {
     $V = ""; // menu_to_1html(menu_find($this->Name, $MenuDepth),0);
+    $V .= '<script src="scripts/jquery-3.2.0.min.js"></script>';
```

Comment:
I don't see another solution. JQuery is loaded too late. Even if I try to add 
<pre>
<script src="scripts/jquery.dataTables.min.js"></script>
<script src="scripts/tools.js"></script>
</pre>
to the `foot.html.twig` tools.js would be loaded too late.